share/machines/Toshiba_HX-20E.xml

```diff
@@ -94,7 +94,7 @@
     <primary slot="3">
```

Comment:
Isn't it more logical to name the lower RAM "Main RAM 1" and the upper part "Main RAM 2"? Or is it the same in other configurations?
Perhaps we should change it the other way around, but then more standardized, like "Main RAM Lower part", "Main RAM Upper part"?
What do you think?

Reply:
Thanks for the updates after our discussion. I think this is addressed now.